config.go

```diff
@@ -174,6 +174,13 @@ func (cfg Config) Build(opts ...Option) (*Logger, error) {
 		return nil, err
 	}
 
+	if cfg.Level == (AtomicLevel{}) {
+		cfg.Level = NewAtomicLevelAt(InfoLevel)
+		if cfg.Development {
+			cfg.Level = NewAtomicLevelAt(DebugLevel)
```

Comment:
Should we return an `error` instead?

If we prefer to add a default behavior for Level when unspecified, we'd probably want to document it on Config.Level.

Reply:
I could see InfoLevel as default in ProductionConfig and DebugLevel in DevelopmentConfig, that's why thought of adding the same defaults here.

Let's return an error then!